### ISSUE TITLE
CI: AttributeError: 'NoneType' object has no attribute 'check_name'

### ISSUE THREAD

**Issue by chloro-pn** [ClickHouse/ClickHouse#66476]
https://github.com/ClickHouse/ClickHouse/actions/runs/9921696378/job/27409993524?pr=66344 
* Run
```
Commit status or Build Report is already present - job will be skipped with status: [success]
```
* Post Run
```
Traceback (most recent call last):
  File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/ci.py", line 1294, in <module>
    sys.exit(main())
  File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/ci.py", line 1252, in main
    job_report.check_name or _get_ext_check_name(args.job_name),
AttributeError: 'NoneType' object has no attribute 'check_name'
Error: Process completed with exit code 1.
```

**Comment by maxknv** [comment]
Hi @chloro-pn thanks for highlighting this.
This issue is fixed https://github.com/ClickHouse/ClickHouse/pull/66471

**Comment by chloro-pn** [comment]
> Hi @chloro-pn thanks for highlighting this. This issue is fixed #66471

That's okay, your work is of great significance and has saved a lot of time. : )
